dev/DeploymentAgent/DeploymentAgent.vcxproj

```diff
@@ -51,7 +51,6 @@
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)'=='Release'" Label="Configuration">
     <UseDebugLibraries>false</UseDebugLibraries>
-    <WholeProgramOptimization>true</WholeProgramOptimization>
     <LinkIncremental>false</LinkIncremental>
```

Comment:
>LinkIncremental [](http://example.com/codeflow?start=28&length=15)

Consider moving this definition to the shared props file.

Reply:
Already did :-)